iOS/Article/ArticleViewController.swift

```diff
@@ -229,6 +230,27 @@ class ArticleViewController: UIViewController, MainControllerIdentifiable {
 		
 	}
 	
+	override func contentScrollView(for edge: NSDirectionalRectEdge) -> UIScrollView? {
+		return currentWebViewController?.webView?.scrollView
+	}
+	
+	func configureNavbar() {
+		let scrollEdge = UINavigationBarAppearance()
+		scrollEdge.configureWithOpaqueBackground()
+		scrollEdge.shadowColor = nil
+		scrollEdge.shadowImage = UIImage()
+		
+		let standard = UINavigationBarAppearance()
+		standard.shadowColor = .opaqueSeparator
+		standard.shadowImage = UIImage()
+		
+		navigationController?.navigationBar.standardAppearance = standard
+		navigationController?.navigationBar.compactAppearance = standard
+		navigationController?.navigationBar.scrollEdgeAppearance = scrollEdge
+		navigationController?.navigationBar.compactScrollEdgeAppearance = scrollEdge
+	}
+	
```

Comment:
Is it possible to move this code to `SceneCoordinator.configureNavigationController` or adjacent to it?

Reply:
Fixed in 90b4199